src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,15 @@
 // your components will all go in this `component` directory.
 // feel free to change this component.js into TodoList.js
+import React, { Component } from 'react';
+
+class TodoList extends Component {
+    render(){
+       return(
+        <div className="container-ListItems">
```

Comment:
You can just make this a functional component, since all you're doing here is rendering the container for todo list items.